app/config/index.js

```diff
@@ -0,0 +1,3 @@
+const registerYourInterestConfig = require('../register-your-interest/config')
```

Comment:
What do you feel about having all app related config-files within one config-folder?

Reply:
Okay if we keep this the same so we can easily delete the register-your-interest folder once we are done with it?